index.js

```diff
@@ -37,6 +41,11 @@ Tree.prototype.walk = function() {
   return this.rootNode.walk()
 };
 
+
+/*
```

Comment:
Could you 🔥  the second blank line here, so there's only one blank line between declarations?